### PR TITLE
Fix typos, and general improvements.

### PR DESCRIPTION
**Pull request by tfrommen**
This PR fixes some typos, and improves both formatting and wording here and there.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 PHP CDS is not a group or organisation, it's a concept. Its core idea is for the community to create RFCs,
 discuss them, then hold a vote on their acceptance as a community standard.
 
-Its **aim is to** be as loosely coupled to any individual, group or organisation as possible.
+Its **aim is** to be as loosely coupled to any individual, group or organisation as possible.
```
Comment (geggleto) [inline README.md:12]: `**aim**`

File: README.md [hunk 2]
```diff
@@ -59,26 +59,26 @@ all discussion should happen in the [php-cds/php-cds-discussion](https://github.
 
 Discussion in a separate repository prevents noise for those who are only interested in voting.
 
-Those who want to be more involved in the community driven standards process can watch the [php-cds/php-cds-discussion](https://github.com/php-cds/php-cds-discussion) repository.
+Those who want to be more involved in the Community-driven Standards process can watch the [php-cds/php-cds-discussion](https://github.com/php-cds/php-cds-discussion) repository.
```
Comment (geggleto) [inline README.md:62]: Community-Driven Standards

File: README.md [hunk 3]
```diff
@@ -59,26 +59,26 @@ all discussion should happen in the [php-cds/php-cds-discussion](https://github.
...
 
 ### Discuss
 
-If you want to get more involved, you can watch the [php-cds/php-cds-discussion](https://github.com/php-cds/php-cds-discussion) repository, where pull-requests and discussion about implementions happen. You can participate on all discussions in this repository.
+If you want to get more involved, you can watch the [php-cds/php-cds-discussion](https://github.com/php-cds/php-cds-discussion) repository, where pull requests and discussions about implementions happen. You can participate in all discussions in this repository.
 
 ### Support
 
-You can express your support for PHP-CDS by pressing the star button. This signals to other people that you support this concept and also that you have an active interest in adopting those community driven standards.
+You can express your support for PHP-CDS by pressing the star button. This signals to other people that you support this concept, and also that you have an active interest in adopting those Community-driven Standards.
```
Comment (geggleto) [inline README.md:81]: Community-Driven Standards

File: README.md [hunk 4]
```diff
@@ -91,31 +91,31 @@ purpose other than to help maintain clean repositories on behalf of the communit
...
-Once you think you have enough positive feedback from the community (as seen by the [Github reactions](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments))
+Once you think you have enough positive feedback from the community (as seen by the [GitHub reactions](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments))
 you are free to open a discussion thread in the php-cds/php-cds-discussion repository. People that are interested in joining
-your RFC can discuss and reply at any time. You can also create Pull-Requests so people can easily comment on proposed code.
+your RFC can discuss and reply at any time. You can also create pull requests so people can easily comment on proposed code.
 
 ## Publish PSR
 
-Once on a RFC has positive community reaction, you have decided to create a discussion and a concrete implemention, and you think your work is
+Once an RFC has positive community reaction, you have decided to create a discussion and a concrete implemention, and you think your work is
```
Comment (geggleto) [inline README.md:113]: Once a RFC

**Comment by geggleto** [comment]
Good work, if you wouldn't mind including the small edits I made on those lines, it would be wonderful!

File: README.md (hunk 1 above)
Comment (tfrommen) [inline README.md:12]: No, `**aim is**` is correct, see the counterpart `**aim is not**` below. I just removed the word _to_ as it is not bold below.

File: README.md (hunk 4 above)
Comment (tfrommen) [inline README.md:113]: _RFC_ is **pronounced** with a starting _a_ so the correct one is _an_ here, no?

File: README.md (hunk 2 above)
Comment (tfrommen) [inline README.md:62]: Title case is with lowercase, but I'm fine with ASP as well.

Comment (OddGreg) [inline README.md:62]: 'Community-Driven Standards' is a proper term representative of an entity. 'community-driven standards' is a descriptive phrase that does not carry the connotation of an entity. It's your choice, but I see no particular reason for proper capitalization.

File: README.md (hunk 4 above)
Comment (marcj) [inline README.md:113]: I'm not a native but I remembered the rule in the way that it's not based on the pronouncement but on the actual first letter. Anyone knows the actual truth™? :D

Comment (xabbuh) [inline README.md:113]: It's based on pronunciation.

Comment (tfrommen) [inline README.md:113]: @marcj it's the other way around. ;)

Even though _URL_ starts with a vowel, it is pronounced _you are ell_, and thus written like "Choose **a** URL" (and not _an URL_).

RFC is the counterpart, starts with no vowel, but is prononunced like it does.

Comment (marcj) [inline README.md:113]: oh ok, thanks for clarification :)

**Comment by marcj** [comment]
Thanks @tfrommen!

**Pull request merged**